docs/content/developer/iota-101/transactions/sign-and-send-transactions.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 description: A guide on how to construct, sign, and submit transactions in a Move-based blockchain.
-tags: [ tutorial, move, rust, typescript, transaction]
+tags: [ tutorial, move, rust, typescript, transaction, rust-sdk]
```

Comment:
This one doesn't have `sdk`, not sure if that matters

Reply:
```suggestion
tags: [ tutorial, sdk, move, rust, typescript, transaction, rust-sdk]
```